_includes/checklist.md

```diff
@@ -1,41 +1,91 @@
-* **Main repository:** <url>
-* **Running libraries:**
-    * <url>
-    * ...
-* **Site:** <url>
-* **Product manager:** @<username>
-* **Technical point of contact:** @<username>
+* **Main repository:** [url]
+* **Site:** [url]
+* **Product manager:** @[username]
+* **System Owner:** @[username of technical point of contact who will be the project representative in the Sprint]
+* **Infrastructure Lead:** @[username]
+* **ATO folder:** [url]
+* **Sprint notes:** [url]
 
 ## TODOs
 
-### Project team
+### ATO Sprint prerequisites
 
-Many of these tasks can be done in parallel. This is just a suggestion of priority.
+#### Infrastructure Lead
 
+- [ ] Set up an ATO intro meeting with the team.
+- [ ] Determine the [impact level](https://pages.18f.gov/before-you-ship/ato/levels/).
```

Comment:
Does this need to be confirmed by anyone? And once determined, do you want it listed here and/or where should it be recorded?